client/dfclient/include/text_creator.hpp

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <ncine/Sprite.h>
+#include "resources.hpp"
+#include "gameplay_state.hpp"
+
+/** Simplifies creating various in-game texts, such as UI messages on player kills.
+ * Designed to allow long-term and one-off text parameter customization. As such,
+ * stores the default values, which are customizable directly or through setter methods
+ * and are supplied if no value for a parameter is given.
+ * 
+ * To simplify creating outlines, this manager also stores the values used to create
+ * the last text. The values are supplied automatically to the outline-creating method,
+ * with the exception of tween easing, which has to be given once more as a parameter.
+ */
+class TextCreator {
```

Comment:
This is a builder pattern, so maybe the `TextBuilder` name would be a little more fitting?

Reply:
I think it's just a flavor :) I felt the TextCreator was just a cooler name, but if you insist it can be changed. I'm just kinda attached to it emotionally, so you need to state it clearly that you want it changed xD :sweat_smile:

Reply:
I'm not insisting too much. I'm fine with leaving it as it is.